docs/_dg/UseCases.md

```diff
@@ -2,11 +2,11 @@
 
 #### Use Case 1: Add Item to Inventory
 
-Preconditions: Item to add does not yet exist in inventory.
+Preconditions: [[ item:Item ]] to add does not yet exist in inventory.
 
-MSS:
+[[ MSS ]]:
```

Comment:
Like OS, this is an invalid key as the glossary entry name is "MSS (Main Success Scenario)". However, I'm going to change this to "Main Success Scenario (MSS)" in another PR, so let's fix it the right way.

```suggestion
[[ main-success-scenario-mss:MSS ]]:
```